kt/entry-generation/godot-entry-generator/src/main/kotlin/godot/entrygenerator/generator/SignalRegistrationGenerator.kt

```diff
@@ -50,7 +50,7 @@ object SignalRegistrationGenerator {
                 add(ClassName(godotRegistrationPackage, GodotKotlinJvmTypes.ktFunctionArgument))
                 add(argumentType.toKtVariantType())
                 add(argumentType.fqName)
-                add(argumentName)
+                add("\"$argumentName\"")
```

Comment:
Please change it here so it's a string and not a literal anymore: https://github.com/utopia-rise/godot-kotlin-jvm/blob/5fb0c6aadd64ed9f06e0706853121a2a845bb4e0/kt/entry-generation/godot-entry-generator/src/main/kotlin/godot/entrygenerator/generator/SignalRegistrationGenerator.kt#L34

With this, the additional quotes are not necessary anymore